ui/src/components/btn-dropdown/QasBtnDropdown.vue

```diff
@@ -107,6 +123,22 @@ export default {
     hasSeparator () {
       return !this.isSmall && this.useSplit
     }
+  },
+
+  watch: {
+    menu: {
```

Comment:
Não seria melhor ter um model chamado `menuOpen` e que você utilize apenas ele como `model`? Pois to vendo que tem um data `isMenuOpened` e uma prop `menu` para controlar o mesmo estado...

Reply:
Respondido discord